### ISSUE TITLE
Pre-trained model shows better performance than paper

### ISSUE THREAD

**Issue by moonsh** [YanFangCS/CyCTR-Pytorch#3]
I am trying to get the same performance as in the paper but it seems better than the paper.

I tested on COCO-20 with resnet50.

Fold 0 ( Split 0) shows 0.4128 accuracy. 
[2022-05-24 14:44:59,046 INFO test.py line 247 1101140] meanIoU---Val result: mIoU 0.4128.
I guess this is correct mIoU for split 0 ? 

It's supposed to be 38.9 based on the paper.

Is there something I missed?
and I wonder what's the meaning of classes in config files.

**Comment by GengDavid** [comment]
Yes, you are right. The reproduced performance is higher than the results reported in the paper. We obtain better results after publishing this paper (with some adjustments to the hyper-parameter). The code is a re-implementation and may not exactly match the results in the paper. 

"classes" is set to 2, which indicates two output channels (foreground and background). This configuration follows PFENet.

**Comment by moonsh** [comment]
@GengDavid   Thank you!!. By the way, can you share the hyper-parameter?  Maybe the hyper-parameter setting is already in the config files?

and..  Have you experienced "WARNING:root:NaN or Inf found in input tensor" this before during training ?

**Comment by GengDavid** [comment]
Yep, the hyper-parameters are up to date.

I met the warning in my experiments at the early stage of the project. Maybe you can try to adjust the learning rate. But this problem can be caused by wide reasons, and I cannot tell you the specific solution.

**Comment by moonsh** [comment]
ah yes thank you!
